src/modules/vtol_att_control/standard.cpp

```diff
@@ -370,8 +378,9 @@ void Standard::fill_actuator_outputs()
 		fw_out[actuator_controls_s::INDEX_PITCH]        = elevon_lock ? 0 : fw_in[actuator_controls_s::INDEX_PITCH];
 		fw_out[actuator_controls_s::INDEX_YAW]          = 0;
 		fw_out[actuator_controls_s::INDEX_THROTTLE]     = _pusher_throttle;
-		fw_out[actuator_controls_s::INDEX_FLAPS]        = 0;
-		fw_out[actuator_controls_s::INDEX_AIRBRAKES]    = 0;
+		fw_out[actuator_controls_s::INDEX_FLAPS]        = _flaps_setpoint_with_slewrate.getState();
```

Comment:
@sfuhrer Where are you resetting these for hover? Do I understand correctly that if you have flaps/spoilers set for backtransition then they will be kept in hover mode?

Reply:
In `void VtolType::update_mc_state()`
`_flaps_setpoint_with_slewrate.update(0.f, _dt);`

Reply:
Side note on this part - don't we want the least amount of drag in hover (so if wind is there it easier to manage)? Meaning doesn't it make more sense to not have anything deflected while in hover?

Reply:
Yes, but we also a don't want a lot of lift when descending, and the spoilers can help with that. The small increase in drag doesn't hurt much I would say.
And keep in mind that with what I propose the spoilers are only deflected when landing, not just in general.